### DISCUSSION TITLE
No music in game

### DISCUSSION THREAD

**Discussion by fluttergust** [The-Wineskin-Project/WineskinServer#333] [Q&A] [answered]
Hi! Hopefully it is alright to ask for some help here.

I've been trying to run this game (Blue Wish Resurrection Plus) on my M2 Mac air + Ventura since version 21.2 - while the sound effects do work, the background music (mp3s) doesn't, even in the latest 22.0.1.

I did find this mentioned on the winehq site ([BWRP](https://appdb.winehq.org/objectManager.php?bShowAll=true&bIsQueue=false&bIsRejected=false&sClass=version&sTitle=&sReturnTo=&iId=40939)) saying that the music requires installing dsound and directmusic, but even after installing those using winetricks the music still doesn't play. (I also tried installing media foundation)

I tested running this with a trial version of crossover 22.0.1 and surprisingly the background music worked there. I'm wondering what I might be missing here? Thanks in advance for any tips!

**Comment by Gcenx** [comment, marked as answer]
Without any logs I can’t really say what might be happening here.

While CrossOver-22.0.1 and WineCX22.0.1 share the same base source I do apply additional patches and enable some additional features that CodeWeavers don’t. Something this helps and sometimes it can cause issues (usually gstreamer wants a plug-in I don’t provide)

**Reply by fluttergust** [reply]
Yes, thought I've forgotten something, attaching the logs here in case its useful! I do see some coreaudio/gstreamer errors but unsure if they are related.

[logs.zip](https://github.com/Gcenx/WineskinServer/files/10223271/logs.zip)

**Reply by Gcenx** [reply]
I won’t be able to look over the log for the moment but if your seeing gstreamer complaining then it might be worth simply disabling it.

Launch winecfg, click the “libraries” tab and add `winegstreamer` then set it to disabled.

That _might_ result in working audio as it will fallback to using other libraries over gstreamer.

**Reply by fluttergust** [reply]
Ahah! After more fiddling around I figured out what was missing... the music played after I installed `quartz`(_feb2010?) via winetricks. Thanks for taking a look though!
